common/src/main/java/com/microsoft/identity/common/internal/ui/webview/challengehandlers/PKeyAuthChallengeHandler.java

```diff
@@ -60,30 +60,35 @@ public PKeyAuthChallengeHandler(@NonNull final WebView view,
     public Void processChallenge(final PKeyAuthChallenge pKeyAuthChallenge) {
         mWebView.stopLoading();
         mChallengeCallback.setPKeyAuthStatus(true);
+        new Thread(new Runnable() {
```

Comment:
Why a new Thread ? ` mWebView.post(new Runnable() { ` is again executed on UI thread anyways right ?